File: WowUp/WowUp/Models/Curse/CurseSearchResult.cs
using System;
using System.Collections.Generic;

namespace WowUp.Models.Curse
{
    public class CurseSearchResult
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public IEnumerable<CurseAuthor> Authors { get; set; }

        public IEnumerable<CurseAttachment> Attachments { get; set; }

        public string WebsiteUrl { get; set; }
        public int GameId { get; set; }
        public int DefaultFileId { get; set; }
        public decimal DownloadCount { get; set; }
        public IEnumerable<CurseFile> LatestFiles { get; set; }
        public IEnumerable<CurseCategory> Catagories { get; set; }
        public int Status { get; set; }
        public int PrimaryCategoryId { get; set; }
        public CurseCategorySection CategorySection { get; set; }
        public string Slug { get; set; }

        public IEnumerable<CurseGameVersionLatestFile> GameVersionLatestFiles { get; set; }

        public bool IsFeatured { get; set; }
        public decimal PopularityScore { get; set; }
        public int GamePopularityRank { get; set; }
        public string PrimaryLanguage { get; set; }
        public string GameSlug { get; set; }
        public string GameName { get; set; }
        public string PortalName { get; set; }
        public DateTime DateModified { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateReleased { get; set; }
        public bool IsAvailable { get; set; }
        public bool IsExperiemental { get; set; }
    }
}

File: wowup-electron/src/app/models/curse/curse-search-result.ts
import { CurseAuthor } from "../../../common/curse/curse-author";
import { CurseAttachment } from "../../../common/curse/curse-attachment";
import { CurseFile } from "../../../common/curse/curse-file";
import { CurseCategory } from "../../../common/curse/curse-category";
import { CurseCategorySection } from "../../../common/curse/curse-category-section";
import { CurseGameVersionLatestFile } from "../../../common/curse/curse-game-version-latest-file";

export interface CurseSearchResult {
  id: number;
  name: string;
  authors: CurseAuthor[];
  attachments: CurseAttachment[];
  websiteUrl: string;
  gameId: number;
  defaultFileId: number;
  downloadCount: number;
  latestFiles: CurseFile[];
  catagories: CurseCategory[];
  status: number;
  primaryCategoryId: number;
  categorySection: CurseCategorySection;
  slug: string;
  gameVersionLatestFiles: CurseGameVersionLatestFile[];
  isFeatured: boolean;
  popularityScore: number;
  gamePopularityRank: number;
  primaryLanguage: string;
  gameSlug: string;
  gameName: string;
  portalName: string;
  dateModified: string;
  dateCreated: string;
  dateReleased: string;
  isAvailable: boolean;
  isExperiemental: boolean;
  summary: string;
}
